src/components/HeroHeader/index.jsx

```diff
@@ -24,25 +25,19 @@ const HeroHeader = ({ t, client, breakpoints: { isMobile } }) => {
   const { data: instanceSettings } = useInstanceSettings(client)
   const publicName = get(instanceSettings, 'public_name', '')
 
+  const showLogout =
+    flag('home.corner.logout') === null ? true : flag('home.corner.logout')
+  const showSettings =
+    flag('home.corner.settings') === null ? false : flag('home.corner.settings')
```

Comment:
We should have the same default behavior since the name is constructed the same way. 

Something like is easier to understand 
`home.corner.logoutIsDisplayed` 
`home.corner.settingsIsDisplayed`

Reply:
Can we have a doc file with all the flags used in the Home app?